env/CMakeLists.txt

```diff
@@ -67,4 +67,24 @@ ExternalProject_Add(
     DEPENDS python-venv
 )
 
+set(STABLE_HLO_VERSION "v1.5.0")
+
+ExternalProject_Add(
+    stablehlo
+    PREFIX ${TTMLIR_TOOLCHAIN_DIR}
+    CMAKE_GENERATOR Ninja
+    CMAKE_ARGS
+      -DCMAKE_BUILD_TYPE=Release
+      -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+      -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+      -DCMAKE_INSTALL_PREFIX=${TTMLIR_TOOLCHAIN_DIR}
+      -DCMAKE_CXX_COMPILER_LAUNCHER=${CMAKE_CXX_COMPILER_LAUNCHER}
+      # TODO: Figure out if we need this (or benefit from it) when it comes to linking stablehlo.
+      #-DSTABLEHLO_BUILD_EMBEDDED=ON
```

Comment:
I saw this commend in their cmake file:
```cmake
# There are 3 build modes, one will be set to ON
#  - Standalone: Build MLIR as a part of StableHLO, requires registering LLVM globals
#  - External: StableHLO built as an external LLVM project (XLA/MHLO uses this)
#  - Embedded: StableHLO built as a part of another MLIR project (torch-mlir uses this)
```

I'm not 100% sure if we want External or Embedded, but we definitely don't want Standalone